manifests/f/freifunkMUC/FreifunkMeet/2024.3.0/freifunkMUC.FreifunkMeet.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: freifunkMUC.FreifunkMeet
+PackageVersion: 2024.3.0
+PackageLocale: en-US
+Publisher: Freifunk Muenchen
+PublisherUrl: https://github.com/freifunkMUC
```

```suggestion
PublisherUrl: https://ffmuc.net
PublisherSupportUrl: https://github.com/freifunkMuc/jitsi-meet-electron/issues
PrivacyUrl: https://ffmuc.net/privacy/
```

Add properties according to @T0biii's suggestion.